app/javascript/articles/components/TagList.jsx

```diff
@@ -1,16 +1,36 @@
 import { h } from 'preact';
 import { tagPropTypes } from '../../common-prop-types';
 
-export const TagList = ({ tags = [] }) => (
-  <div className="crayons-story__tags">
-    {tags.map((tag) => (
-      <a className="crayons-tag" href={`/t/${tag}`}>
-        <span className="crayons-tag__prefix">#</span>
-        {tag}
-      </a>
-    ))}
-  </div>
-);
+export const TagList = ({ tags = [], flare_tag }) => {
+  let tagsToDisplay = tags;
+  if (flare_tag) {
+    tagsToDisplay = tagsToDisplay.filter((tag) => tag !== flare_tag.name);
+  }
+  return (
+    <div className="crayons-story__tags">
+      {flare_tag && (
+        <a className="crayons-tag" href={`/t/${flare_tag.name}`}>
+          <span
+            className="crayons-story__flare-tag"
```

Comment:
Similarly to what I suggested in [previous comment](https://github.com/forem/forem/pull/10528/files#diff-419ded8bb2b41fca12af2061c1272094), we should drop `.crayons-stroy__flare-tag`